docs/administration/private-cloud-migration/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Private cloud migration
+description: Guidelines for migrating an on-premises Octopus instance to private cloud hosting
+position: 15
+hideInThisSectionHeader: true
+---
+
+Teams looking to migrate their on-premises Octopus instance to private cloud hosting must take multiple factors into account to ensure a smooth migration. This document provides a guide to ensure teams successfully migrate their instance with minimal interruption to their deployments.
+
+## Checklist
+
+There are a number of factors to consider when migrating an on-premises instance to cloud hosting:
+
+* What version of Octopus are you running?
+* Who is using the on-premises Octopus instance?
+* How long can Octopus be offline for before it interrupts critical operations?
+* Who can test each project to ensure it deploys correctly after the migration?
+* Do you require a continuous audit history after the migration?
+* How do users authenticate with Octopus?
+* Where have tentacles been installed?
+* What kind of tentacles have been configured (polling or listening)?
+* Are there any firewall rules restricting traffic to tentacles?
+* Do you want to host Octopus in a Linux container on a platform like Kubernetes or ECS?
+* Do you have a direct network connection from your cloud provider to your on-premises infrastructure?
+* Where are your packages stored?
+* Do you have any CI servers integrated with Octopus?
+* Do you have any subscriptions configured in Octopus?
+* Do you have any external tools or scripts that call the Octopus API?
+* Do you have external scripts or CI servers using API keys?
+
+
+### What version of Octopus are you running?
+
+Some of the migration options require a relatively recent version of Octopus to be installed. For this reason the first step in any migration is to update the on-premises instance to the latest version of Octopus.
+
+### Who is using the on-premises Octopus instance?
+
+Most teams looking to move their on-premises Octopus instance to a private cloud tend to have multiple teams deploying multiple projects through Octopus. How the migration is performed is largely dictated by the requirements of each of the impacted teams. It is therefore critical to understand which teams are using Octopus and what applications they are deploying. Having this information allows you to answer the next questions.
```

Comment:
It's probably worth noting that if existing teams have integrations (API/CLI scripts etc), written against the on-prem instance, this will need to be updated to move to a new server. 

_This was a problem for a customer that had dozens of teams, some who had written tons of custom scripts against their projects. They needed to budget time to review and update if they moved instances/spaces._